cime_config/tests.py

```diff
@@ -650,6 +650,7 @@
             "SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.scream-output-preset-3",
             "ERP_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.scream-output-preset-4",
             "ERS_D_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.scream-rad_frequency_2--scream-output-preset-5",
+            "ERS_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.scream-small_kernels--scream-output-preset-5",
```

Comment:
Did you mean to use `small_kernels_p3` (or shoc) maybe? To test the single sk choice?

Reply:
It's ok to have 3 testmods. I was hoping you would test one of those two in this PR, so that we can check (in logs) that indeed we use it in, say, shoc but not p3...